### DISCUSSION TITLE
How come most of the 'local' channels are just news?

### DISCUSSION THREAD

**Discussion by scumola** [iptv-org/community#278] [Q&A] [answered]
Going through most of the 'local' channels (ABC/CBS/NBC), they're news-only channels.  Not much different from the ABCN/CBSN/... channels.  Can these be fixed?  No actual 'shows' like the evening prime-time shows can be seen on any of the 'local' channels - just news.

**Comment by Central302** [comment]
If you Simply Need to Prime Time Program, Find Channels That Using Network's Logo. in The Playlist.
e.x. For NBC Network Programing, Use WBTS-CD  (East Coast) or KING-DT (West Coast).
Notice That Find to Full Local Channel Streaming Are Difficult.

**Comment by benevenstanciano** [comment]
Many network affiliate stations stream their news 24/7 online, because they don't have the rights to stream the network or syndicated shows.

**Comment by MapGuy11** [comment, marked as answer]
We are working on it but you have to think you normally pay for cable or satellite for these channels so getting them for free is quite hard. Yes, you could make the case that OTA is free well to transcode to H264 and IP costs money. Please remember that.
